airflow/contrib/hooks/sagemaker_hook.py

```diff
@@ -59,9 +59,11 @@ def check_for_url(self, s3url):
         if not s3hook.check_for_bucket(bucket_name=bucket):
             raise AirflowException(
                 "The input S3 Bucket {} does not exist ".format(bucket))
-        if not s3hook.check_for_key(key=key, bucket_name=bucket):
-            raise AirflowException("The input S3 Key {} does not exist in the Bucket"
-                                   .format(s3url, bucket))
+        if not s3hook.check_for_key(key=key, bucket_name=bucket)\
```

Comment:
Add some comments here. Telling readers that a valid s3 input is either a file object(single data file) or a prefix with file objects(group of data files).

Reply:
added comment